tests/wdio/Data/Setup.js

```diff
@@ -0,0 +1,59 @@
+const Page = require('../vaah-webdriverio/Page');
+
+class Setup extends Page{
+    constructor() {
+        super();
+        this.params = {
+            group: {
+                count: null,
+                name: null,
+            }
+        }
+        this.element = {
+            install_button: '//span[text()= "Install"]'
```

Comment:
@we-abhijeet-g001 try to add `data-testid`

Reply:
data-testid was not an attribute of the install button. Used element by text selector.

Reply:
@we-abhijeet-g001 can you add tasks for @we-vineet-k001 to add those `data-testid`